### DISCUSSION TITLE
Service Remove and/or stop is not available in package

### DISCUSSION THREAD

**Discussion by Daxcor69** [apocas/dockerode#684] [Q&A] [answered]
I must be missing something, but I can find a method to remove a created service nor just stop the service.  This service is just one container, so am I just supposed to use the  container methods? The docker api does have  "Delete Service" spec.

```
await containerId.stop();
await containerId.remove({ force: true });
```

Thanks
Brad

**Comment by apocas** [comment, marked as answer]
https://github.com/apocas/dockerode/blob/master/lib/service.js
